src/Umbraco.Web.BackOffice/Extensions/UmbracoBackOfficeServiceCollectionExtensions.cs

```diff
@@ -23,6 +23,10 @@ public static void AddUmbracoBackOffice(this IServiceCollection services)
         {
             services.AddAntiforgery();
 
+            //We need to have runtime compilation of views when using backoffice. We could consider having only this when a specific config is set.
+            //But as far as I can see, there are still precompiled views, even when this is activated, so maybe it is okay.
+            services.AddControllersWithViews().AddRazorRuntimeCompilation();
```

Comment:
I don't think we want `AddControllersWithViews` here since we're also calling `AddMvc()` which calls this underneath as well. IIRC calling this method in some orders i think has implications. IMO for now we should just keep all AddMvc() stuff in one spot until we look more into this so we should just hvae `.AddRazorRuntimeCompilation();` as part of our single AddMvc call for now.